tools/submission/submission_checker.py

```diff
@@ -1166,10 +1166,10 @@
     "F1": r"^{[\"\']exact_match[\"\']\:\s*[\d\.]+,\s*[\"\']f1[\"\']\:\s*([\d\.]+)}",
     "WER": r"Word Error Rate\:.*, accuracy=([0-9\.]+)%",
     "DICE": r"Accuracy\:\s*mean\s*=\s*([\d\.]+).*",
-    "ROUGE1": r"^ROUGE1=([\d\.]+).*",
-    "ROUGE2": r"^ROUGE2=([\d\.]+).*",
-    "ROUGEL": r"^ROUGEL=([\d\.]+).*",
-    "GEN_LEN": r"^GEN_LEN=([\d\.]+).*",
+    "ROUGE1": r"'rouge1':\s([\d.]+).*",
+    "ROUGE2": r"'rouge2':\s([\d.]+).*",
+    "ROUGEL": r"'rougeLsum':\s([\d.]+).*",
+    "GEN_LEN": r"'gen_len':\s([\d.]+).*",
```

Comment:
I had to add a ".*" to the beginning of the 4 regular expressions and then get the expected results as follows. 
```
.*'rouge1':\s([\d.]+).* 42.556635 "Acc matched:" 43.178
.*'rouge2':\s([\d.]+).* 19.922265 "Acc matched:" 19.8027
[2023-07-15 21:56:09,852 submission_checker.py:1597 WARNING] open/cTuning/results/amd_zen4_workstation-reference-gpu-pytorch-v1.13.0-default_config/gptj-99/offline/accuracy accuracy not met: expected=19.922265, found=19.8027
.*'rougeLsum':\s([\d.]+).* 29.688219 "Acc matched:" 40.3119
.*'gen_len':\s([\d.]+).* 3615190.2 "Acc matched:" 4162985
```